### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by pixunil**
grafik

### CODE REVIEW

File: evap/evaluation/templates/contact_modal.html [hunk 1]
```diff
@@ -48,6 +48,8 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
     function {{ modal_id }}Show() {
         $('#{{ modal_id }}ActionButton').unbind().click(function(){ {{ modal_id }}Action(); });
         $('#{{ modal_id }}').modal();
+        // remove wrongly added padding by Bootstrap
```
Comment (janno42) [inline evap/evaluation/templates/contact_modal.html]: can you please add a comment where exactly (which file, which line) bootstraps adds the padding?

File: evap/static/css/evap.scss [hunk 2]
```diff
@@ -141,6 +141,19 @@ a.btn:not([href]):not(.disabled).btn-dark {
     display: inline-flex;
     background-color: lighten($light-gray, 2%);
     border-radius: 0.2rem;
+
```
Comment (janno42) [inline evap/static/css/evap.scss]: please remove empty lines here and below

File: evap/static/css/evap.scss [hunk 3]
```diff
@@ -141,6 +141,19 @@ a.btn:not([href]):not(.disabled).btn-dark {
...
+    .btn-toolbar {
+        background: $white;
+    }
+
+    .btn-group {
+        background: lighten($light-gray, 2%);
+
+        &::before {
+            width: $spacer * 0.5;
```
Comment (janno42) [inline evap/static/css/evap.scss]: please use `width: 0.5rem;`

File: evap/evaluation/templates/contact_modal.html [hunk 4]
```diff
@@ -48,6 +48,8 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
     function {{ modal_id }}Show() {
         $('#{{ modal_id }}ActionButton').unbind().click(function(){ {{ modal_id }}Action(); });
         $('#{{ modal_id }}').modal();
+        // remove wrongly added padding by Bootstrap (https://github.com/twbs/bootstrap/blob/v4-dev/js/src/modal.js#L429)
+        $('body, .navbar, .footer').css('padding-right', '');
```
Comment (karyon) [inline evap/evaluation/templates/contact_modal.html]: if you think that is a bug, please consider reporting it. there have been previous (closed) reports of this (e.g. [here](https://github.com/twbs/bootstrap/issues/24934)), but still, please consider it.

Comment (janno42) [inline evap/evaluation/templates/contact_modal.html]: you can remove the `.footer` now, it's not fixed anymore.

**Comment by janno42** [comment]
unfortunately you're breaking the language button now. can you please make sure that the background color won't be changed there?

File: evap/evaluation/templates/contact_modal.html (hunk 4 above)
Comment (pixunil) [reply evap/evaluation/templates/contact_modal.html]: I think it is no bug on their side, the `padding` just does not play well with a `overflow: scroll !important` on the body, so I think it is our fault here

Comment (pixunil) [reply evap/evaluation/templates/contact_modal.html]: Changed to a more redundant, but future-proof solution  (same selectors like bootstrap)

**Pull request merged**